website/README.md

```diff
@@ -63,4 +63,4 @@ npm run serve
 
 
 ## Maintainers
-OneTable Community
+XTable Community
```

Comment:
add a link to https://incubator.apache.org/projects/xtable.html ?

Reply:
Good idea, added the link